<img src=http://www.acofi.edu.co/eiei2016/wp-content/uploads/2016/09/Logo-Universidad-ICESI.jpg width="300" height="200" align="center">

#**<big><big>Matemáticas Aplicadas II</big></big><br><small>Departamento de Ciencias Físicas y Aplicadas/ Facultad de Ingeniería, Diseño y Ciencias Aplicadas</small><br><small>Notebook2: Teorema Fundamental del Cálculo - Métodos de Integración - Integración Numérica. </small>**  
##**Profesor Luis Fernando Azcárate Mesa**

1. **Lectura sugerida del texto de Sloughter. Calculus from Approximation to Theory (2020)**: Sección  4.3 Teorema fundamental del cálculo y 4.4 Usando el Teorema fundamental del cálculo

2. **Lectura sugerida del texto de Lax. Calculus With Applications (2014)**: Sección 6.3 Teorema fundamental del cálculo.



# Desarrollado por:

- David Dulce A00398802

- Rodolfo Moreno A00395543

- Joshua Lattke A00372620

# Taller en Clase

A continuación encontrará una serie de ejercicios y problemas que deben ser desarrollados en los equipos de trabajo establecidos. Los procesos análiticos deben ser escritos en formato LaTex. Deben ser muy claros al indicar los códigos de Python utilizados. Todas las respuestas y concluisones obtenidas de la solución deben ser explícitas.

In [ ]:
import numpy as np
import sympy as sp
import matplotlib.pyplot as plt
import IPython.display as IP
import math
import random

# Ejercicio 1
1.5 Puntos

Evalúen cada una de las siguientes integrales. Hagan explícitos procesos analíticos que los conducen a la respuesta. Para cada caso, verifiquen sus respuestas usando el método de aproximación de Simpson.

In [ ]:
def simpson(f, a, b, n=1000):
  dx = (b - a) / n
  x = np.linspace(a, b, n+1)
  y = f(x)
  sum = np.sum(y[0:-1:2] + 4*y[1::2] + y[2::2])
  return sum * dx/3

## a

$ \displaystyle \int_{0}^{1} \frac {e^{2x}-1}{e^{x}} \, dx $

$$ \int_{0}^{1}\frac{e^{2x}-1}{e^{x}}dx $$

$$ \int_{0}^{1} \frac{e^{2x}}{e^{x}} - \frac{1}{e^{x}}dx $$

$$ \int_{0}^{1} e^{2x-x} - \frac{1}{e^{x}}dx $$

$$ \int_{0}^{1} e^{x} - \int_{0}^{1}\frac{1}{e^{x}}dx $$

$$ \int_{0}^{1} e^{x} - \int_{0}^{1}\frac{1}{e^{x}}dx $$

$$ \Big |^{1}_0e^{x}-\Big|^{1}_0-{e^{-x}} $$

$$ \Big|^{1}_0e^{x}-\Big|^{1}_0\frac{-1}{e^{x}} $$

$$ e^{1}-e^{0}-(\frac{-1}{e^{1}}-(\frac{-1}{e^{0}})) $$

$$ e-1-(\frac{-1}{e}-(\frac{-1}{1})) $$

$$ e-1-(\frac{-1}{e}-(-1)) $$

$$ e-1-(\frac{-1}{e}+1) $$

$$ e-1-\frac{-1}{e}-1 $$

$$ e-\frac{-1}{e}-2 $$

$$ \simeq 1.08616 $$

In [ ]:
simpson(lambda x: ( np.exp(2*x) - 1) / np.exp(x), 0, 1)

1.0861612696304936

Junto al codigo de python se confirma que la integral evaluada de 0 a 1 es 1.08616

##b

$\displaystyle\int_{-2}^{2}\left | x^{2}-1 \right |dx$   

Primero debemos saber en que espacios $x$ es negativa

$\left | x\right |=-x,x < 0$
 y
$x,x\ge0 $

$ x^{2}-1< 0$

$ x^{2}<-1$

$ -1< x< 1$

Ahora sabemos que x es negativa en los intervalos de -1 a 1

Despues de poner los nuevos intervalos para evaluar la integral queda asi:

$\int_{1}^{2}x^{2}-1dx+\int _{-1}^{1}-(x^{2}-1)dx+\int_{-2}^{-1}x^{2}-1dx$

$\int_{1}^{2}x^{2}-1dx+\int _{-1}^{1}-x^{2}+1dx+\int_{-2}^{-1}x^{2}-1dx$   

Ahora ya podemos evaluar

$\int_{1}^{2}\frac{x^{3}}{3}-xdx+\int _{-1}^{1}\frac{-x^{3}}{3}+xdx+\int_{-2}^{-1}\frac{x^{3}}{3}-xdx$

$\Big|^{2}_1\frac{x^{3}}{3}-x+\Big|^{1}_{-1}\frac{-x^{3}}{3}+x+\Big|^{-1}_{-2}\frac{x^{3}}{3}-x$

$\frac{2^{3}}{3}-2-(\frac{1^{3}}{3}-1)+(
-\frac{1^{3}}{3}+1-(-\frac{(-1)^{3}}{3}+(-1)))+(
\frac{(-1)^{3}}{3}-(-1)-(\frac{(-2)^{3}}{3}-(-2)))$  

$\frac{8}{3}-2-(\frac{1}{3}-1)+(
-\frac{1}{3}+1-(-\frac{(-1)}{3}+(-1)))+(
\frac{-1}{3}-(-1)-(\frac{(-8)}{3}-(-2)))$


$\frac{8}{3}-2-(\frac{1}{3}-1)+(
-\frac{1}{3}+1-(-\frac{(-1)}{3}+(-1)))+(
\frac{-1}{3}+1-(\frac{(-8)}{3}-(-2)))$


$\frac{2}{3}-(\frac{-2}{3})+
\frac{2}{3}-(-\frac{-2}{3})+
\frac{2}{3}-(\frac{-2}{3})$

$\frac{2}{3}+\frac{2}{3}+
\frac{2}{3}+\frac{2}{3}+
\frac{2}{3}+\frac{2}{3}$

$\frac{12}{3}$  

$4$

In [ ]:
simpson(lambda x: abs(x**2 - 1),-2,2)

4.0

Junto al codigo de python se confirma que la integral evaluada de -2 a 2 es 4

##c

$\displaystyle\int_{1}^{4}\frac{dx}{\sqrt{9x^{3}}}$   

$\int_{1}^{4}\frac{1}{\sqrt{9x^{3}}}dx$

$\int_{1}^{4}\frac{1}{3\sqrt{x^{3}}}dx$

$\int_{1}^{4}\frac{1}{3}\frac{1}{3\sqrt{x^{3}}}dx$

$\frac{1}{3}\int_{1}^{4}\frac{1}{\sqrt{x^{3}}}dx$

$\frac{1}{3}\int_{1}^{4}\frac{1}{\sqrt{x^{3}}}dx$

$\frac{1}{3}\int_{1}^{4}\frac{1}{x^{\frac{3}{2}}}dx$

$\frac{1}{3}\int_{1}^{4}-\frac{2}{\sqrt{x}}dx$

$\frac{1}{3}\Big|_1^4-\frac{2}{\sqrt{4}}-(-\frac{2}{\sqrt{1}})$

$\frac{1}{3}(-\frac{2}{2}-(-\frac{2}{1}))$

$\frac{1}{3}(-1-(-2))$

$\frac{1}{3}(-1+2)$

$\frac{1}{3}(1)$

$\frac{1}{3}$

In [ ]:
simpson(lambda x: 1/((9*(x)**3)**(0.5)),1,4)

0.3333333333352982

Junto al codigo de python se confirma que la integral evaluada de 1 a 4 es 0.3'

# Ejercicio 2
1 Punto



Encuentre la aproximación lineal a la función $\displaystyle\ f(x)=\int_{2}^{x}\sqrt[3]{u-1}du$ para $x=3$

primero calculamos la antiderivdad de $\sqrt[3]{u-1}du$


$\displaystyle\ f(u)=\int\sqrt[3]{u-1}du$

para simplificar realizamos sustitucion $v=u-1$ entonces $dv=du$,y la integral se convierte en:

$∫v^{{1}/{3}}dv$

integramos

$f(u)=\frac{3}{4}v^{{1}/{3}}+C$

reemplazamos $v=u-1$

$f(u)=\frac{3}{4}(u-1)^{{4}/{3}}+C$

la funcion $f(x)$ es la antiderivadad evaluada en x:
$f(u)=\frac{3}{4}(x-1)^{{4}/{3}}+C$

para encontrar $C$, usamos la condicion $f(2)=0$, esto nos representa el area de bajo la curva desde 2 hasta 2, lo cual es cero:

$0=\frac{3}{4}(2-1)^{{4}/{3}}+C$

$0=\frac{3}{4}(1)^{{4}/{3}}+C$

$0=\frac{3}{4}+C$

$C=-\frac{3}{4}$

la funcion $f(x)$ es:
$f(u)=\frac{3}{4}(x-1)^{{4}/{3}}-\frac{3}{4}$

ahora derivamos $f(x)$ para encontrar $f'(x)$

$f'(x)=\frac{3}{4}*\frac{4}{3}(x-1)^{{1}/{3}}*1$

$f'(x)=(x-1)^{{1}/{3}}$

la aproximacion lineal en x=3 sera la ecuacion de la recta tangente $f(x)$ en este punto usando la formula punto-pendiente:

$y-y1=m(x-x_{1})$


Donde $(x_1,y_1)=(3,f(3))$ y $m=f'(3):$

$m=f'(3)=(3-1)^{{1}{3}}=2^{{1}{3}}$

$f(3)=\frac{3}{4}(3-1)^{{4}{3}}-\frac{3}{4}=\frac{3}{4}(2)^{{4}/{3}}-\frac{3}{4}$

$f(3)=\frac{3}{4}(2)^{{4}/{3}}-\frac{3}{4}=\frac{3}{4}(2)^{{4}/{3}}-\frac{3}{4}={3}{4}(2^{{4}/{3}}-1)$

Entonces. $(x_1,y_1)=(3,\frac{3}{4}(2^{{4}/{3}}-1)).$

por tanto la ecuciacion recta tange es:

$y-\frac{3}{4}(2^{{4}/{3}}-1)=2^{{1}/{3}}(x-3)$

simplificando la aproximacion lineal:

$y=2^{{1}/{3}}(x-3)+\frac{3}{4}(2^{{4}/{3}}-1)$

Esta es la aproximacion lineal de $f(x)$ alrededor de $x=3$






In [ ]:
# Definir la variable simbólica y la función f(x)
x = sp.Symbol('x')
f = sp.integrate(sp.root(x - 1, 3), (x, 2, x))

# Calcular la derivada de f(x) y evaluarla en x = 3
f_prime = f.diff(x)
f_prime_at_3 = f_prime.subs(x, 3)

# Calcular f(3)
f_at_3 = f.subs(x, 3)

# Construir la ecuación de la recta tangente y(x) = mx + c
m = f_prime_at_3
c = f_at_3 - m * 3

# Mostrar la aproximación lineal
print("La aproximación lineal de f(x) alrededor de x = 3 es:")
print(f"y(x) = {m} * (x - 3) + {c}")


La aproximación lineal de f(x) alrededor de x = 3 es:
y(x) = 2**(1/3) * (x - 3) + -3*2**(1/3)/2 - 3/4


# Ejercicio 3
1.5 Puntos

Consideren la función $ f(x) = xsenx $ definida en el intervalo $ \left [ 0,4 \pi \right ] $ y sea $ \displaystyle\ g(x)=\int_{0}^{x} (tsent) dt $.
Determinen los valores de $x$ para los cuales la función $g(x)$ alcanza sus valores mínimo y máximo en el intervalo $ \left [ 0, 4 \pi \right ] $.

## Derivada

La función $ g(x) $ se define como:

$$ g(x) = \int_{0}^{x} t \sin t \, dt $$

Para encontrar $ g'(x) $, aplicamos el teorema fundamental del cálculo parte 1:

$$ g'(x) = x \sin x $$

## Puntos críticos

Luego hay que encontrar los puntos críticos donde la derivada es igual a cero:

$$ x \sin x = 0 $$

Esto sucede para $ x = 0, \pi, 2 \pi, 3 \pi $.

## Subintervalos

Podemos dividir el intervalo original en cuatro subintervalos, en los que evaluaremos si $ g(x) $ es creciente o decreciente mediante el [criterio de la primera derivada](https://es.wikipedia.org/wiki/Criterio_de_la_primera_derivada):

| Subintervalo | $ g'(x) $ | monotonía |
| - | - | - |
| $ [0,π] $ | $ g'(x) > 0 $ | creciente |
| $ [π,2π] $ | $ g'(x) < 0 $ | decreciente |
| $ [2π,3π] $ | $ g'(x) > 0 $ | creciente |
| $ [3π,4π]$ | $ g'(x) < 0 $ | decreciente |

## Puntos máximo y mínimo

- máximo: $ [ \pi, \pi^2 ] $

- mínimo: $ [ 3 \pi, 9 \pi^2 ] $

# Ejercicio 4
1 Punto



Suponga que la velocidad (descendente) de un paracaidista está dada por la función $ v(t) = 30(1-e^{-t}) \frac {m}{s} $ durante los primeros 5 segundos de un salto. Calcule la distancia que ha caido el paracaidista durante este tiempo.

**Nota: recuerden que la velocidad es la razón de cambio instantánea, respecto al tiempo, de la función distancia**.  

Hallar la distancia recorrida ($\vec x$) en el instante en que $ t = 5 $ segundos.

**Asumiendo que el paracaidista cae en un vacío ($ F_d = 0 N$) y desde el reposo ($ v_0 = 0 \frac {m}{s} $)**:

$$ v(t) = 30(1-e^{-t}) $$

$$ \therefore v(t) = \frac {d}{dt} {\vec x} $$

Por teorema fundamental del cálculo parte 1:

$$ x(t) = \int \vec v \, dt $$

$$ x(t) = \int 30(1-e^{-t}) \, dt $$

$$ x(t) = 30 \int (1-e^{-t}) \, dt $$

$$ x(t) = 30 \left ( \int 1 \, dt - \int e^{-t} \, dt \right ) $$

$$ x(t) = 30( t - e^{-t} ) + C $$

$$ x(t) = 30( t - e^{-t} ) + 0 $$

$$ x(t) = 30( t - e^{-t} ) $$

Para $ t = 5 $ segundos:

$$ x(5) = 30( 5 - e^{-5} ) $$

In [ ]:
x = 30 * (5 - np.exp(-5))
print(x)

149.79786159002742


$$ x \approx 149.7978 $$

$$ x \approx 149.8 m $$

Asumiendo que el paracaidista cae en un vacío ($ F_d = 0 N$) y desde el reposo ($ v_0 = 0 \frac {m}{s} $), entonces la distancia que ha caído el paracaidista en el instante $t = 5s$ son **$149.8m$ aproximadamente**.